opentelemetry/opentelemetry.cabal

```diff
@@ -50,14 +50,15 @@ library
   build-depends:
     base >= 4.13 && < 5,
     bytestring,
+    text,
```

Comment:
The `opentelemetry` sublibrary intentionally depends on as little libraries as possible, preferably only on things that GHC itself depends on. I think we can get by without depending on `text`.

Reply:
Resolved in 7231325